warehouse/upload.go

```diff
@@ -1309,6 +1300,12 @@ func (job *UploadJobT) setUploadError(statusError error, state string) (newstate
 	} else {
 		errorByState["attempt"] = 1
 	}
+	// check if all errors in upload are skip errors
+	hasOnlySkippedErrors := false
```

Comment:
I think you can use job.hasAllTablesSkipped instead of even having to do a multierror type check right?

Reply:
Yes, makes sense 👍